rust/src/lib.rs

```diff
@@ -35,6 +35,7 @@ pub mod json;
 pub mod applayer;
 pub mod filecontainer;
 pub mod filetracker;
+#[macro_use] pub mod parser;
```

Comment:
minor: I think it's nicer to have the 'pub mod parser;' on a new line

Reply:
done